### PR TITLE
Percentiles in Perf Analysis

### PR DESCRIPTION
**Pull request by ShekharGupta1988**
Two changes:
1. Update views for percentile view for performance
2. Enable TCP connection analysis

### CODE REVIEW

File: AppLensV2/app/Site/SiteCtrl.ts [hunk 1]
```diff
@@ -163,13 +167,17 @@ module SupportCenter {
...
+                        item.disabled = true;
+                    }
+
                     self.latencyChartData.push(item);
                 });
             }, function (err) {
                 self.perfDataLoading = false;
                 self.ErrorHandlerService.showError(ErrorModelBuilder.Build(err));
-            });
+                });
```
Comment (stevenernst131) [inline AppLensV2/app/Site/SiteCtrl.ts]: is this indented correctly? hard to tell on here

File: AppLensV2/app/Site/SiteCtrl.ts [hunk 2]
```diff
@@ -163,13 +167,17 @@ module SupportCenter {
                 var requestsIterator = 0;
 
                 _.each(chartDataList, function (item: any) {
-                    item.color = DetectorViewHelper.runtimeAvailabilityColors[iterator];
+                    item.color = DetectorViewHelper.defaultColors[iterator++];
+                    if (item.key === '90th Percentile' || item.key === '95th Percentile') {
+                        item.disabled = true;
```
Comment (stevenernst131) [inline AppLensV2/app/Site/SiteCtrl.ts]: will this mean only 50th percentile will show up?

File: AppLensV2/app/Site/SiteCtrl.ts (hunk 1 above)
Comment (ShekharGupta1988) [reply AppLensV2/app/Site/SiteCtrl.ts]: ya it is :). visual studio did the indentation.

File: AppLensV2/app/Site/SiteCtrl.ts (hunk 2 above)
Comment (ShekharGupta1988) [reply AppLensV2/app/Site/SiteCtrl.ts]: no. 90th and 95th will also show up but unselected

**Comment by ShekharGupta1988** [comment]
@stevenernst131  - approved?

**Pull request merged**